contents/blog/product-engineer-vs-software-engineer.md

```diff
@@ -13,15 +9,17 @@ category: Engineering
 tags:
   - Explainers
   - Product engineers
+crosspost:
+  - Product engineers
 ---
 
-Software companies are changing. Where once product managers and software engineers dominated, now a new role is emerging: the [product engineer](/blog/what-is-a-product-engineer).
+Software companies were once dominated by two types of roles: product managers and software engineers, but this is changing. A new role is emerging: the [product engineer](/blog/what-is-a-product-engineer).
 
 Want proof? Look at this graph:
 
 ![product-engineer-trends](../images/blog/product-engineer-trend.png)
 
-Online searches for product engineers have grown over 80% in the last year and are 3x greater now than just two years ago.
+Online searches for product engineers saw a big and sustained jump in 2022.
```

Comment:
```suggestion
Online searches for product engineers have grown by 80% since 2021 as companies, particularly startups like ourselves, [incident.io](http://incident.io/),  and [Ghost](https://ghost.org/), prioritize engineers who are customer obsessed, talk to them directly, and comfortable working autonomously. 
```